src/main/treeDataProviders/dependenciesTree/buildsDataProvider.ts

```diff
@@ -48,6 +49,7 @@ export class BuildsDataProvider implements vscode.TreeDataProvider<DependenciesT
             ScanUtils.setScanInProgress(true);
             const credentialsSet: boolean = this._treesManager.connectionManager.areAllCredentialsSet();
             this._treesManager.logManager.logMessage('Starting to load builds details...', 'INFO');
+            await this.sendUsageReport();
```

Comment:
I think to maximize the performance we should send usage report asynchronously, as we do in the JFrog CLI.
```suggestion
            this.sendUsageReport();
```